lib/livebook/migration.ex

```diff
@@ -188,4 +111,20 @@ defmodule Livebook.Migration do
 
     Map.put(file, :file_system_type, file_system_type)
   end
+
+  defp v1_remove_old_filesystems() do
+    # For a while we had a migration converting this to the new table.
+    # Now we just delete the old entries for simplicity. If someone
+    # upgrateds from an old version, they just need to re-add their
+    # file systems. This has negligible impact.
+    for id <- Storage.all(:filesystem), do: Storage.delete(:filesystem, id)
+  end
+
+  defp v1_remove_default_file_system_id() do
+    # For a while we had a migration converting this to default dir.
+    # Now we just delete the old setting for simplicity. If someone
+    # upgrateds from an old version, they just need to set the default
+    # fiel system again. This has negligible impact.
+    Storage.delete_key(:settings, "global", :default_file_system_id)
+  end
```

Comment:
I removed the migrations as per the TODOs, but keep the deletion as per the other comment :)

Reply:
I believe the idea for removing the TODOs is that we would no longer migrate earlier versions than 1. So if we found a database without version, we would just discard it and start from scratch. However, the current patch can get a very old version and we will dirty state into it, which may cause us trouble in the far away future?

So if we want to remove the TODOs, I would do something like:

```
    storage_version =
      case Storage.fetch_key(:system, "global", :migration_version) do
        {:ok, version} -> version
        :error -> :ets.delete_all_objects(tab); 0
      end
```

at the very top of the init function. WDYT?

Reply:
@josevalim clearing the storage is if we want to get rid of all v1 migrations entirely, but the current migrations are compatible with any prior version, so I think they are worth keeping. The two migrations had TODOs because we changed table names, they were mostly a convenience so that people don't loose the config, but without these migrations the old tables are just ignored, which is fine. The reason I kept an explicit delete is to make sure there are no leftover tables, in case someone upgrades from an older version. Does that make sense?

Reply:
Perfect ship it